components/central-application-gateway/cmd/applicationgateway/options.go

```diff
@@ -2,53 +2,54 @@ package main
-	applicationSecretsNamespace string
-	requestTimeout              int
 	proxyTimeout                int
-	requestLogging              bool
-	proxyCacheTTL               int
-	kubeConfig                  string
-	apiServerURL                string
+	requestTimeout              int
 }
 
-func parseArgs() *options {
-	externalAPIPort := flag.Int("externalAPIPort", 8081, "External API port.")
-	proxyPort := flag.Int("proxyPort", 8080, "Proxy port for Kyma OS or MPS bundles with a single API definition")
-	proxyPortCompass := flag.Int("proxyPortCompass", 8082, "Proxy port for Kyma MPS.")
-	applicationSecretsNamespace := flag.String("applicationSecretsNamespace", "kyma-system", "Namespace where Application secrets used by the Application Gateway exist")
-	requestTimeout := flag.Int("requestTimeout", 1, "Timeout for services.")
-	proxyTimeout := flag.Int("proxyTimeout", 10, "Timeout for proxy call.")
-	requestLogging := flag.Bool("requestLogging", false, "Flag for logging incoming requests.")
-	proxyCacheTTL := flag.Int("proxyCacheTTL", 120, "TTL, in seconds, for proxy cache of Remote API information")
-	kubeConfig := flag.String("kubeConfig", "", "Path to a kubeconfig. Only required if out-of-cluster.")
-	apiServerURL := flag.String("apiServerURL", "", "The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
+func parseArgs(log *zap.Logger) (opts options) {
+	flag.StringVar(&opts.apiServerURL, "apiServerURL", "", "The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
+	flag.StringVar(&opts.applicationSecretsNamespace, "applicationSecretsNamespace", "kyma-system", "Namespace where Application secrets used by the Application Gateway exist")
+	flag.IntVar(&opts.externalAPIPort, "externalAPIPort", 8081, "Port that exposes the API which allows checking the component status and exposes log configuration")
+	flag.StringVar(&opts.kubeConfig, "kubeConfig", "", "Path to a kubeconfig. Only required if out-of-cluster")
+	opts.logLevel = zap.LevelFlag("logLevel", zap.InfoLevel, "Log level: panic | fatal | error | warn | info | debug. Can't be lower than info")
+	flag.IntVar(&opts.proxyCacheTTL, "proxyCacheTTL", 120, "TTL, in seconds, for proxy cache of Remote API information")
+	flag.IntVar(&opts.proxyPort, "proxyPort", 8080, "Port that acts as a proxy for the calls from services and Functions to an external solution in the default standalone mode or Compass bundles with a single API definition")
+	flag.IntVar(&opts.proxyPortCompass, "proxyPortCompass", 8082, "Port that acts as a proxy for the calls from services and Functions to an external solution in the Compass mode")
+	flag.IntVar(&opts.proxyTimeout, "proxyTimeout", 10, "Timeout for requests sent through the proxy, expressed in seconds")
+	flag.IntVar(&opts.requestTimeout, "requestTimeout", 1, "Timeout for requests sent through Central Application Gateway, expressed in seconds")
```

Comment:
Could you unify the timeouts? The default value for `requestTimeout` in Helm values is set to 10 seconds.